docs/example_navigation_ros_codegen.md

```diff
@@ -0,0 +1,527 @@
+---
+id: example_navigation_ros_codegen
+title: Navigation of ground vehicle using ROS codegen
+description: Nonlinear model predictive control with OpEn using ROS
+---
+
+<script type="text/x-mathjax-config">MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});</script>
+<script type="text/javascript" async src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+
+Example contributed by Guido Sanchez ([Github](https://github.com/gmsanchez/))
+
```

Comment:
Add some introductory text including information about:
- Husky (including some photos)
- What this example is about (MPC for Husky, position/pose tracking, obstacle avoidance etc)
- Do you have a video? It could be good to start with a demonstration of what the end result looks like